aws/resource_aws_ecs_service.go

```diff
@@ -481,9 +494,8 @@ func resourceAwsEcsServiceUpdate(d *schema.ResourceData, meta interface{}) error
 		}
 	}
 
-	if d.HasChange("network_configration") {
-		input.NetworkConfiguration = expandEcsNetworkConfigration(d.Get("network_configuration").([]interface{}))
-	}
+	//d.HasChange("network_configration") is not working, so explicity calling method.
```

Comment:
Not sure why the the func HadChange is not detecting a change on "network_configuration".  Maybe we need to implement equals on the structure?